### ISSUE TITLE
Training on iPhone data

### ISSUE THREAD

**Issue by mtaktash** [FangjinhuaWang/PatchmatchNet#77]
Hi! I have a dataset of iPhone multiview images with depth information and I want to train PatchmatchNet on this dataset. However, the network converges to a not very sharp depth maps on this dataset or doesn't converge to meaningful results at all. 

I think the issue might be in depth min and max (on dtu it's 425 to 935, on my dataset it's from 0.03 to roughly 26), and I'm using default PatchmatchNet parameters, so something might be off. Can you help to understand if this might be an issue and guide me with better parameters values? I'm using colmap camera poses if this might help.

I'll also appreciate any comments on how to train on iPhone depth data if you have tried this

**Comment by FangjinhuaWang** [comment]
Hi,

The depth range should not be a problem since depth is normalized in the network. Besides, evaluation datasets like ETH3D also have very large depth range (from about 20cm to 20m). How do you do view selection, depth range decision? You can use colmap2input.py to do all of them on the sparse reconstruction you get from COLMAP. For the advice, maybe you can try to finetune the pretrained model on DTU.

**Comment by mtaktash** [comment]
Yes, I use colmap preprocessing with my network and I also tried to finetune DTU model on my dataset, it doesn't work still. To be honest, all MVS nets don't work on this dataset for some unknown reason

**Comment by mtaktash** [comment]
I've seen that evaluation datasets have very large depth range, so now not sure what is the issue...

**Comment by FangjinhuaWang** [comment]
Since 'all MVS nets don't work on this dataset for some unknown reason', I guess the problem may be from the dataset. How do you get your ground truth depth? Is it perfectly aligned with your RGB camera?
